README.md

```diff
@@ -1,2 +1,87 @@
-# protocol-integration-tests
-Integration tests for protocol backend repositories
+# Protocol Integration Tests
+
+This is a set of tests that can be run against the backend services and infrastructure that make up Kiva Protocol to
+verify that a variety of basic use cases are functional.
+
+## Components Under Test
+
+The services tested are:
+- [Gateway](https://github.com/kiva/protocol-gateway)
+- [Aries Key Guardian](https://github.com/kiva/aries-key-guardian)
+- [Aries Guardianship Agency](https://github.com/kiva/aries-guardianship-agency)
+- [Bio Auth Service](https://github.com/kiva/guardian-bio-auth/tree/main/bio_auth_service)
+- [Bioanalyzer Service](https://github.com/kiva/guardian-bio-auth/tree/main/bioanalyzer_service)
+- [Demo Controller](https://github.com/kiva/protocol-demo)
+- FSP Controller (Note: The code for this service is in a private repository. We'll make it public soon!)
```

Comment:
fyi:  does not need to change the README unless theres other updates:  there is also the TDC controller.  The transaction APIs in FSP controller and AGA will make calls to the TDC.  At this time, I do not believe any of our integration tests actually test any of the transaction API.